.circleci/config.yml

```diff
@@ -99,6 +99,42 @@ jobs:
       - store_test_results:
           path: /go/out/tests
 
+  e2e-simple-v1alpha3-v2:
```

Comment:
How about:
- call it 'e2e-simple'
- remove the old e2e-simple, which is using v1

We need to remove the old one either way.